src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+public class FileReaderServiceImpl implements ReaderService {
+    private final BufferedReader bufferedReader;
+
+    public FileReaderServiceImpl(BufferedReader bufferedReader) {
+        this.bufferedReader = bufferedReader;
+    }
```

Comment:
Class BufferedReader is not a stateless reusable service class, you have to create it the method